### PR TITLE
Provide a way in console to access orphaned builds / deployments

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin/issues/6485

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

File: assets/app/views/builds.html [hunk 1]
```diff
@@ -51,11 +51,8 @@
                   <!-- We only show the first build from a build config for now-->
                   <tr ng-repeat="build in buildConfigBuilds | orderObjectsByDate : true | limitTo : 1" ng-if="buildConfigs[buildConfigName] || !unfilteredBuildConfigs[buildConfigName]">
                     <td data-title="Name">
-                      <a ng-if="buildConfigs[buildConfigName]" href="{{buildConfigName | navigateResourceURL : 'BuildConfig' : projectName}}">{{buildConfigName}}</a>
-                      <span ng-if="buildConfigs && !buildConfigs[buildConfigName] && buildConfigName != ''">
-                        {{buildConfigName}}
-                        <span class="pficon pficon-warning-triangle-o" data-toggle="tooltip" data-placement="right" title="This build config no longer exists" style="cursor: help;"></span>
-                      </span>
+                      <a ng-if="buildConfigName != ''" href="{{buildConfigName | navigateResourceURL : 'BuildConfig' : projectName}}">{{buildConfigName}}</a>
```
Comment (spadgett) [inline assets/app/views/builds.html]: `ng-if="buildConfigName"`?

File: assets/app/views/deployments.html [hunk 2]
```diff
@@ -51,14 +51,9 @@
...
-                        {{deploymentConfigName}}
-                        <span class="pficon pficon-warning-triangle-o" data-toggle="tooltip" data-placement="right" title="This deployment config no longer exists" style="cursor: help;"></span>
-                      </span>
+                      <a ng-if="deploymentConfigName != ''" href="{{deploymentConfigName | navigateResourceURL : 'DeploymentConfig' : deployment.metadata.namespace}}">{{deploymentConfigName}}</a>
+                      <span ng-if="deploymentConfigs[deploymentConfigName].status.details.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-placement="right" data-trigger="hover" dynamic-content="{{deploymentConfigs[deploymentConfigName].status.details.message}}"></span>
+                      <span ng-if="deploymentConfigs && !deploymentConfigs[deploymentConfigName] && deploymentConfigName != ''" class="pficon pficon-warning-triangle-o" data-toggle="tooltip" data-placement="right" title="This deployment config no longer exists" style="cursor: help;"></span>
```
Comment (spadgett) [inline assets/app/views/deployments.html]: Same comment as above.

**Comment by spadgett** [comment]
One nit, otherwise LGTM

**Comment by jwforres** [comment]
cleaned up other places where we were doing the `!= ''` check, take a quick look?

**Comment by jwforres** [comment]
just deployments.html and builds.html changed

**Comment by jwforres** [comment]
[merge]

**Comment by spadgett** [comment]
[merge]

**Pull request merged**